cpp/python/libcudf_cffi/tests/test_errorhandling.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+from libcudf_cffi import ffi, libcudf  # , GDFError
+
+from libcudf_cffi.tests.utils import new_column, get_dtype
+
+
+def test_cuda_error():
+    dtype = np.float32
+
+    col = new_column()
+    gdf_dtype = get_dtype(dtype)
+
+    libcudf.gdf_column_view(col, ffi.NULL, ffi.NULL, 0, gdf_dtype)
+
+    # with pytest.raises(GDFError) as raises:
+    #     libcudf.gdf_add_generic(col, col, col)
+
+    # raises.match("CUDA ERROR.")
```

Comment:
What's the goal of this test?

Reply:
@kkraus14 : This is existing test case only ! Probably after renaming the path from `libgdf_cffi` to `libcudf_cffi`, it is showing differently.  We can see there is another `test_errorhandling.py` being `marked` as deleted also.
 
So, there is nothing change as functionality is concern - Like all the places in code drop `libcudf_cffi` and `libcudf` is updated to pick up the intended module name. Thank you !